base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x, T)
+    mod(x, T)
+    %(x, T)
+
+For an integer type `T`, finds `z::T` such that `x` ≡ `z` (mod n), where n is the
+cardinality of `T` and `z` is restricted to the interval of values representable by `T`.
```

Comment:
is cardinality of a type a term we expect people to understand without any further elaboration?